src/Apps/Order/Utils/__tests__/orderUtils.jest.ts

```diff
@@ -21,32 +21,32 @@ describe("order utils", () => {
       })
     })
 
-    describe("order payment is invalid", () => {
-      it("returns US_BANK_ACCOUNT if available payment methods includes it", () => {
+    describe("when currently selected order payment payment is invalid (paymentSet is false)", () => {
```

Comment:
Took me a while to remember what is this `paymentSet` is about and how it correlates with `paymentMethod`. So tried to update descriptions to make it a bit clearer.